Python/Product/Uwp/Templates/Projects/BackgroundService/PythonBackgroundService.pyproj

```diff
@@ -55,4 +55,9 @@
 
   <Import Project="$(PtvsTargetsFile)" />
 
+  <PropertyGroup>
+    <TargetPlatformVersion>$winsdkversion$</TargetPlatformVersion>
+    <TargetPlatformMinVersion>$winsdkminversion$</TargetPlatformMinVersion>
+  </PropertyGroup>
+
```

Comment:
This is intentional. The version property is actually set to a default value ("7.0") by one of the VS .props or .targets files the python UWP .targets file is importing. I haven't dug into this yet but I didn't want to mess around with changing that for now.